From 13a0096830525949a2ad22a7623f7711915f6382 Mon Sep 17 00:00:00 2001
From: kas User <kas@example.com>
Date: Sun, 20 Apr 2025 04:38:08 +0000
Subject: [PATCH] Don't add _INCLUDEDIR variables to include_directories

For some reason, these variables end up always pointing to /usr/include
for bitbake-supplied libraries. This is an egregious hack to allow the
software to build anyways.
---
 cmake/ProcessPackage.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/ProcessPackage.cmake b/cmake/ProcessPackage.cmake
index 316662568..1dabf8db8 100644
--- a/cmake/ProcessPackage.cmake
+++ b/cmake/ProcessPackage.cmake
@@ -61,7 +61,7 @@ MACRO(ProcessPackage PKG_NAME)
             SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${_arg}")
         ENDFOREACH(_arg ${${_XPREFIX}_LDFLAGS_OTHER})
         LIST(APPEND ${PKG_LIB_OUTPUT} "${${_XPREFIX}_LIBRARIES}")
-        INCLUDE_DIRECTORIES(${${_XPREFIX}_INCLUDEDIR})
+	# INCLUDE_DIRECTORIES(${${_XPREFIX}_INCLUDEDIR})
     ELSE()
         IF(NOT ${PKG_NAME}_GUESSED)
             # Try some more heuristic
@@ -124,4 +124,4 @@ MACRO(ProcessPackage PKG_NAME)
 
     UNSET(_lib CACHE)
     UNSET(_incl CACHE)
-ENDMACRO(ProcessPackage name)
\ No newline at end of file
+ENDMACRO(ProcessPackage name)
-- 
2.47.2

